lib/rubocop/cop/style/unnecessary_arguments.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for unnecessary arguments to builtin methods
```

Comment:
I'd expand on the description here (and ideally go beyond builtin methods with some form of configuration).